dependency/actions/compile/entrypoint/main.go

```diff
@@ -0,0 +1,311 @@
+package main
+
+import (
+  _ "embed"
+	"errors"
+  "flag"
+  "fmt"
+  "github.com/paketo-buildpacks/packit/v2/fs"
+  "log"
+  "os"
+  "path/filepath"
+	"strings"
+
+	"github.com/paketo-buildpacks/php-dist/actions/compile/entrypoint/internal"
+)
+
+func main() {
+  var (
+    version   string
+    outputDir string
+    target    string
+  )
+
+  flag.StringVar(&version, "version", "", "PHP dependency version")
+  flag.StringVar(&outputDir, "outputDir", "", "Output directory for PHP dependency artifact")
+  flag.StringVar(&target, "target", "", "Dependency target")
+  flag.Parse()
+
+  if version == "" {
+	  fail(errors.New(`missing required input "version"`))
+  }
+
+  if outputDir == "" {
+	  fail(errors.New(`missing required input "outputDir"`))
+  }
+
+  if target == "" {
+	  fail(errors.New(`missing required input "target"`))
+  }
+
+  dependencyDir, err := os.MkdirTemp("", "php-dist")
+  if err != nil {
+	  log.Fatal(err)
+  }
+
+	err = installLibuv()
+  if err != nil {
+	  log.Fatal(err)
+  }
+
+	err = runSymlinkCommands()
+  if err != nil {
+	  log.Fatal(err)
+  }
+
+  err = installPHP(version, dependencyDir)
+  if err != nil {
+	  log.Fatal(err)
+  }
+
+	var extFile string
+	if strings.Contains(version, "8.0") {
+    extFile = "/tmp/entrypoint/extensions/extensions-8.1.json"
```

Comment:
I might be missing something but why is this path hardcoded to `/tmp`? Where does that path come from?

Reply:
@sophiewigmore The Dockerfile places this in /tmp.../
Do you think it makes more sense to just copy to the working dir?